phir/model.py

```diff
@@ -81,16 +139,38 @@ class QOp(Op):
 class COp(Op):
     """Classical operation."""
 
-    cop: str
+    # From https://github.com/CQCL/phir/blob/main/spec.md#table-i---cop-assignment-arithmetic-comparison--bitwise-operations
+    cop: Literal[
```

Comment:
Curious to see what Ciaran says here, but I think there's some amount of desire to leave `Cop` extensible so that other classical oracles can be used with Pecos for use in evaluating classical operations.

Reply:
But I guess it's fine for the pydantic spec to align with what works out of the box right now.

Reply:
Yeah, I am unsure of what's optimal here. More validation requires more restrictions. Certainly, will wait until hearing from Ciaran before merging this PR.